web/elm/src/Pipeline.elm

```diff
@@ -355,8 +351,22 @@ viewGroupsBar model =
     Html.ul [ class groupClass ] groupList
 
 
-viewGroup : List String -> String -> Concourse.PipelineGroup -> Html Msg
-viewGroup selectedGroups url grp =
+viewGroup :
+    { a
+        | selectedGroups : List String
+        , pipelineLocator : Concourse.PipelineIdentifier
+    }
+    -> Concourse.PipelineGroup
+    -> Html Msg
+viewGroup { selectedGroups, pipelineLocator } grp =
+    let
+        url =
+            Routes.Pipeline
```

Comment:
This seems... not as good as delegating to Routes.pipelineRoute like we used to.